app.js

```diff
@@ -1,19 +1,21 @@
 import { CronJob } from 'cron';
 import { app, errorHandler } from 'mu';
 import fs from 'fs-extra';
-import request from 'request';
+//import request from 'request';
 import loadSources from './data-sources';
 import publish from './publication';
 import { loadTasksFromConfig } from './task';
 import { RUN_ON_STARTUP, LOAD_EXTERNAL_SQL_SOURCES } from './config/env';
+import fetch from 'node-fetch';
 
 const tasks = loadTasksFromConfig();
 
 /** Schedule cron job */
 const cronFrequency = process.env.CRON_PATTERN || '0 0 2 * * *';
 new CronJob(cronFrequency, function() {
   console.log(`Data conversion triggered by cron job at ${new Date().toISOString()}`);
-  request.post('http://localhost/conversion-tasks');
+  fetch('http://localhost/conversion-tasks', {method: 'POST', body: null});
```

Comment:
Is `body` required to be passed if it's just `null`?

Reply:
What I saw in the documentation was the usage of body: null.